ocs_ci/utility/utils.py

```diff
@@ -2915,29 +2915,100 @@ def get_zone_number(self):
         return prev
 
 
-def convert_device_size(unformatted_size, units_to_covert_to):
+def convert_device_size(unformatted_size, units_to_covert_to, convert_size=1000):
     """
     Convert a string representing a size to an int according to the given units
     to convert to
 
     Args:
         unformatted_size (str): The size to convert (i.e, '1Gi'/'100Mi')
         units_to_covert_to (str): The units to convert the size to (i.e, TB/GB/MB)
+        convert_size (int): set convert by 1000 or 1024
 
     Returns:
         int: The converted size
 
     """
     units = unformatted_size[-2:]
     abso = int(unformatted_size[:-2])
-    conversion = {
-        "TB": {"Ti": abso, "Gi": abso / 1000, "Mi": abso / 1e6, "Ki": abso / 1e9},
-        "GB": {"Ti": abso * 1000, "Gi": abso, "Mi": abso / 1000, "Ki": abso / 1e6},
-        "MB": {"Ti": abso * 1e6, "Gi": abso * 1000, "Mi": abso, "Ki": abso / 1000},
-        "KB": {"Ti": abso * 1e9, "Gi": abso * 1e6, "Mi": abso * 1000, "Ki": abso},
-        "B": {"Ti": abso * 1e12, "Gi": abso * 1e9, "Mi": abso * 1e6, "Ki": abso * 1000},
+    conversion_1000 = {
+        "TB": {
+            "Ti": abso,
+            "Gi": abso / 1000,
+            "Mi": abso / 1e6,
+            "Ki": abso / 1e9,
+            "Bi": abso / 1e12,
+        },
+        "GB": {
+            "Ti": abso * 1000,
+            "Gi": abso,
+            "Mi": abso / 1000,
+            "Ki": abso / 1e6,
+            "Bi": abso / 1000,
+        },
+        "MB": {
+            "Ti": abso * 1e6,
+            "Gi": abso * 1000,
+            "Mi": abso,
+            "Ki": abso / 1000,
+            "Bi": abso / 1e6,
+        },
+        "KB": {
+            "Ti": abso * 1e9,
+            "Gi": abso * 1e6,
+            "Mi": abso * 1000,
+            "Ki": abso,
+            "Bi": abso / 1000,
+        },
+        "Bi": {
+            "Ti": abso * 1e12,
+            "Gi": abso * 1e9,
+            "Mi": abso * 1e6,
+            "Ki": abso * 1000,
+            "Bi": abso,
+        },
     }
-    return conversion[units_to_covert_to][units]
+    conversion_1024 = {
```

Comment:
I know that was my idea in the comments of alerts to do it with 2 dicts, but I think with could manage with 1 dict. have the convert_size =1 or = 1.024 and multiple it in one dict.

Reply:
Something like this. Then we will maintain only one dict.
    conversion_all = {
        "TB": {
            "Ti": abso * convert_size,
            "Gi": abso / 1000 * convert_size,
            "Mi": abso / 1e6 * convert_size,
            "Ki": abso / 1e9 * convert_size,
            "Bi": abso / 1e12 * convert_size,
        },

Reply:
`In [5]: 1024**2
Out[5]: 1048576

In [6]: 1.024*1e6
Out[6]: 1024000.0`